Getting-Started/Code/Source-Control/index.md

```diff
@@ -0,0 +1,139 @@
+
+**Note:**
+There is a file /app_data/packages/installed/installedPackages.config that lists all packages installed via the Umbraco backoffice, depending on how you update or install packages to your site, it can be useful to track changes to this file in source control
+
+#### Umbraco Models Builder
+
+The strategy here will depend a little on which mode ['Umbraco Models Builder'](../../../reference/templating/modelsbuilder) is running in.
+ - **Pure Live**, no source control allow Umbraco to generate via the backoffice
+ - **App_Data**, you can source control the generated files, including them within your Visual Studio Solution means they will be built into your application's dll
+ - **Dll**, if your working with a team, and commiting the generated dll, then this can be problematic managing merge conflicts, so remember to 'regenerate' your dll, after pulling from source control... but probably you want to use API...
+ - **API**, generate the models into a seperate extension project and source control the files from there.
+
+#### Media
+
+The Media section of Umbraco (unless configured otherwise) stores files in the /media folder, these can be updated by editors, in the Umbraco backoffice, so generally speaking you would not source control these files.
+
+#### Packages and Plugins
+
+The **App_Plugins** folder is the home for all third party packages installed on your site
+Depending on how you installed the plugin - Nuget vs The Backoffice, how you deploy your site to your production environment, and whether the plugin requires site specific configuration... will affect how you choose to version control a particular third party plugin. 
+
+** Nuget + Buildserver - if the plugin is installed via NuGet then as long as the packages.config file in the root of your project is source controlled, then the installed files for individual plugins shouldn't need to be source controlled (and your deployment process should pull the packages implementation files from Nuget during the deployment process).
+** Nuget + Manual deploy - here you don't necessarily need to source control the files, but you do need to remember to deploy them.
+** Backoffice - if your working with other developers, then it may be easier to add the plugin files to source control, or at least communicate to them that the particular package needs be installed via the backoffice
+** Backoffice + Buildserver - you'll need to include the plugin files in source control as the build server won't know to restore them - if the plugin/package doesn't come with a NuGet resource, consider setting up your own local NuGet repository for your build server to pull the files from.
+
+**Note**
+Each plugin could be different depending on what it's functionality is, and so may contain files it would be useful to track via Source control, and ones that should be ignored, check with the plugin's supporting website/developer for more information.
```

Comment:
> it's 

Teach me some English... Isn't it "on what _its_ functionality is" as it is a possessive thing?